src/Chainweb/Rosetta/Internal.hs

```diff
@@ -121,6 +121,7 @@ matchLogs typ bh logs coinbase txs
   | coinV2Upgrade v cid bheight = matchRemediation (upgradeTransactions v cid)
   | to20ChainRebalance v cid bheight = matchRemediation (twentyChainUpgradeTransactions v cid)
   | pact4coin3Upgrade At v bheight = matchRemediation coinV3Transactions
+  | chainweb214Pact At v bheight = matchRemediation coinV4Transactions
```

Comment:
what does this do?

Reply:
Honoring @LindaOrtega's note here: https://github.com/kadena-io/chainweb-node/blob/d62076ee60d31e500a6abeaab30abccc7455d708/src/Chainweb/Pact/Transactions/UpgradeTransactions.hs#L74